tests/thread/host/spinlock_test_host.cpp

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <errno.h>
+#include <openenclave/host.h>
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/tests.h>
+#include <pthread.h>
+#include <signal.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <cassert>
+#include <chrono>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include "thread_u.h"
+
+#define ERRNO_TEST_TIMES 1
+#define LARGE_ENOUGH_MEMORY_SIZE 1024 * 1024 * 1024
+#define LONG_ENOUGH_TIME 40000
+
+int g_flag_withlock = 0;
+static uint32_t g_lock = 0;
+static int g_error_num = 0;
+
+int _wait_single(void* thread_id, int time_out)
```

Comment:
could you chance this name to "timeout_ms" to make it clear that the timeout is in milliseconds?